Cargo.toml

```diff
@@ -28,24 +28,25 @@ panic = 'abort'     # Abort on panic
-lightning-block-sync = { version = "0.0.125", features = ["rpc-client", "tokio"] }
-lightning-transaction-sync = { version = "0.0.125", features = ["esplora-async-https", "time"] }
-lightning-liquidity = { version = "0.1.0-alpha.6", features = ["std"] }
-
-#lightning = { git = "https://github.com/lightningdevkit/rust-lightning", branch="main", features = ["std"] }
-#lightning-invoice = { git = "https://github.com/lightningdevkit/rust-lightning", branch="main" }
-#lightning-net-tokio = { git = "https://github.com/lightningdevkit/rust-lightning", branch="main" }
-#lightning-persister = { git = "https://github.com/lightningdevkit/rust-lightning", branch="main" }
-#lightning-background-processor = { git = "https://github.com/lightningdevkit/rust-lightning", branch="main", features = ["futures"] }
-#lightning-rapid-gossip-sync = { git = "https://github.com/lightningdevkit/rust-lightning", branch="main" }
-#lightning-transaction-sync = { git = "https://github.com/lightningdevkit/rust-lightning", branch="main", features = ["esplora-async"] }
-#lightning-liquidity = { git = "https://github.com/lightningdevkit/lightning-liquidity", branch="main", features = ["std"] }
+# lightning = { version = "0.0.125", features = ["std"] }
+# lightning-invoice = { version = "0.32.0" }
+# lightning-net-tokio = { version = "0.0.125" }
+# lightning-persister = { version = "0.0.125" }
+# lightning-background-processor = { version = "0.0.125", features = ["futures"] }
+# lightning-rapid-gossip-sync = { version = "0.0.125" }
+# lightning-block-sync = { version = "0.0.125", features = ["rpc-client", "tokio"] }
+# lightning-transaction-sync = { version = "0.0.125", features = ["esplora-async-https", "time"] }
+# lightning-liquidity = { version = "0.1.0-alpha.6", features = ["std"] }
+
+lightning = { git = "https://github.com/cequals/rust-lightning", branch="amackillop/0.0.125-patched", features = ["std"] }
+lightning-invoice = { git = "https://github.com/cequals/rust-lightning", branch="amackillop/0.0.125-patched" }
+lightning-net-tokio = { git = "https://github.com/cequals/rust-lightning", branch="amackillop/0.0.125-patched" }
+lightning-persister = { git = "https://github.com/cequals/rust-lightning", branch="amackillop/0.0.125-patched" }
+lightning-background-processor = { git = "https://github.com/cequals/rust-lightning", branch="amackillop/0.0.125-patched", features = ["futures"] }
+lightning-rapid-gossip-sync = { git = "https://github.com/cequals/rust-lightning", branch="amackillop/0.0.125-patched" }
+lightning-block-sync = { git = "https://github.com/cequals/rust-lightning", branch="amackillop/0.0.125-patched", features = ["rpc-client", "tokio"] }
+lightning-transaction-sync = { git = "https://github.com/cequals/rust-lightning", branch="amackillop/0.0.125-patched", features = ["esplora-async-https", "time"] }
+lightning-liquidity = { git = "https://github.com/cequals/lightning-liquidity", branch="amackillop/v0.1.0-alpha.6-patched", features = ["std"] }
```

Comment:
Unfortunately, just using a patch doesn't seem to work with cargo chef so I'm using branches here instead. This branch just backports a small change to 0.0.125 to add Debug impl's to some structs